apps/api/src/app/inbox/dtos/update-all-notifications-request.dto.ts

```diff
@@ -1,8 +1,6 @@
 import { IsArray, IsOptional, IsString } from 'class-validator';
 
-import type { NotificationFilter } from '../utils/types';
-
-export class UpdateAllNotificationsRequestDto implements Pick<NotificationFilter, 'tags'> {
+export class UpdateAllNotificationsRequestDto {
```

Comment:
`Pick`, `Omit` and other helper generic types are also incompatible with Nest.js' SWC metadata.